src/Compilers/Core/CodeAnalysisTest/EditorConfigTests.cs

```diff
@@ -262,50 +262,50 @@ public void ReservedKeys()
         public void SimpleNameMatch()
         {
             string regex = EditorConfig.TryCompileSectionNameToRegEx("abc");
-            Assert.Equal("^abc$", regex);
+            Assert.Equal("^.*/abc$", regex);
 
-            Assert.Matches(regex, "abc");
-            Assert.DoesNotMatch(regex, "aabc");
-            Assert.DoesNotMatch(regex, " abc");
-            Assert.DoesNotMatch(regex, "cabc");
+            Assert.Matches(regex, "/abc");
```

Comment:
❓ Where do we ensure the path always has a `/` and isn't just a file name?

Reply:
https://github.com/dotnet/roslyn/pull/27396/files#diff-a969f68cd2bc892509eb83bec00d060eR320

It's always expected that the EditorConfig parser is provided an absolute file path, but I don't Assert that. Do you think that would be helpful?

Reply:
Just added a check to throw if the file path is not an absolute path.